### PR TITLE
Add websocketpp

### PR DESCRIPTION
**Pull request by nirbheek**
header-only c++ dependency

### CODE REVIEW

**Comment by eli-schwartz** [comment]
The wrap should be added to releases.json.

What about adding the examples/tests behind an option (and using it in CI)?

**Comment by nirbheek** [comment]
> The wrap should be added to releases.json.

Thanks, I knew I was forgetting something :)

> What about adding the examples/tests behind an option (and using it in CI)?

What's the point of doing that? It's just a fancy `include_directories:`. This sort of thing should probably be handled by the wrap system automatically via special `[provide]` syntax even.

**Comment by eli-schwartz** [comment]
For example, people using the wrap could then test that websocketpp itself doesn't have bugs on their system or compiler.

**Comment by nirbheek** [comment]
I looked at that, and it's more work than I am willing to spend on this right now. I'm happy to let someone else complete it if they want to.

File: releases.json [hunk 1]
```diff
@@ -2343,6 +2343,14 @@
       "1.24-1"
     ]
   },
+  "websocketpp": {
+    "dependency_names": [
+      "websocketpp"
+    ],
+    "versions": [
+      "0.8.2-0"
```
Comment (eli-schwartz) [inline releases.json:2351]:
```suggestion
      "0.8.2-1"
```

It's the first published wrap for this websocketpp release.

Comment (nirbheek) [reply releases.json:2351]: Ah, fdk-aac and a couple of other wraps use `-0`. Fixed.

Comment (eli-schwartz) [reply releases.json]: It's a very wishy-washy thing to begin with. I just tend to view "0" as meaning "locally testing this, will bump for publication", so it looks off. :D

**Pull request merged**